api/handlers/containerhandler.py

```diff
@@ -240,7 +232,7 @@ def get_jobs(self, cid):
 
         permchecker(noop)('GET', cid)
 
-        analyses = cont.get('analyses', [])
+        analyses = list(config.db.analyses.find({'parent.type': 'session', 'parent.id': cont['_id']}).sort('created', -1))
```

Comment:
Can we use `AnalysisStorage().get_analyses(<>)` here like above? It doesn't always happen, but I try to not directly make db calls from container handlers because then we can put calls that should always happen in the `storage` layer and feel relatively confident they will always be called. Things like compliance calculation, default field filling, etc.